src/ride/controllers/create_ride_controller.py

```diff
@@ -51,13 +51,18 @@ def process(self):
         creator_person = Person.objects.get(id=creator)
 
         # Create path and ride and return new ride with given fields
-        path = Path.objects.create(
-            start_location_place_id=start_location_place_id,
-            start_location_name=start_location_name,
-            end_location_place_id=end_location_place_id,
-            end_location_name=end_location_name,
-        )
-        path.save()
+        path_exists = Path.objects.filter(start_location_place_id=start_location_place_id,end_location_place_id=end_location_place_id,).exists()
```

Comment:
see https://github.com/cuappdev/scoop-backend/pull/14/files#r983891760

we will no longer be using this `.exists()` pattern and `.get()` if existing since we are not working with large querysets in these CREATE controllers but rather a queryset that has either 0 or 1 elements.